lib/teleterm/api/proto/v1/cluster.proto

```diff
@@ -33,6 +33,8 @@ message Cluster {
   bool leaf = 5;
   // User is the cluster access control list of the logged-in user
   LoggedInUser logged_in_user = 7;
+  // features describes the auth servers features
+  Features features = 8;
```

Comment:
Any reason not to include the entire ping response? It seems like knowing things like the version, whether or not it's a FIPS server, and if there are any license warnings might also be useful.

Reply:
No reason besides wanting to keep the API concise and tidy to what is needed for this PR. Not opposed to adding in the extra, unused responses to prevent extra PRs in the future.

Reply:
License warnings, server version number, etc, are located on a different ping response. The `features` key here is from the `authClient` ping response where as things like server version, license warnings, etc etc seem to come from the `webClient` ping response and are currently ignored [here](https://github.com/gravitational/teleport/blob/master/lib/teleterm/clusters/storage.go#L153-L158). Those values 

I agree those values would be valuable to have, but not on the `features` key.